CHANGELOG.md

```diff
@@ -1135,12 +1135,91 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Fix missing CodeMirror styles on Firefox and Safari.
 - Fix Safari crashes in `<mwc-tab>` code.
 
-## [0.1.0-pre.2] - 2020-09-12
+## 0.1.0-pre.2 - 2020-09-12
```

Comment:
Ah - I see that there isn't anything to link here, and same with the initial release. Nice!